planetarium/NineChronicles
# Fix abnormal VFX position bugs

### Description

1. In the past, VFX of battle result are rendered above the Help UI.
2. Fixed VFX.cs, add bool field `isLayerInitializedWithChild`. Default value is `true`.
3. If that is false, initialize children particle systems like root particle system.

### How to test

1. Eneter any stage
2. Open help Popup in Battle result UI
3. Check above the Help UI

### Related Links

[[UI] 스테이지 결과 UI 의 도움말 팝업에서 별 이펙트가 노출되는 현상](https://app.asana.com/0/1141562434100787/1197211236908617/f)

### Screenshot

before
![210806_help popup ui-vfx-bug](https://user-images.githubusercontent.com/48484989/128463548-f8716aaf-170e-4537-bbd7-d15a25cb10f9.gif)

after
![210806_vfx above ui](https://user-images.githubusercontent.com/48484989/128471666-a0040e8d-8afa-4c66-b567-e641fcf648d6.gif)
